.github/workflows/codecov.yaml

```diff
@@ -14,8 +14,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Setup Sui CLI and install dependencies
-        uses: ./.github/actions/install
+      - name: Setup Sui CLI
+        uses: ./.github/actions/install-sui-cli
+        with:
+          sui-version: "$(jq -r '.SUI_VERSION' version.json)"
+
+      - name: Setup Node Dependencies
+        uses: ./.github/actions/install-node-dependencies
+        with:
+          node-version: 18
```

Comment:
Default it to 18 in the action. It's only meant to be used in this repo anyways so avoids additional configuration